examples/image_classifier/streaming_video_client_side_batching/custom_handler.py

```diff
@@ -0,0 +1,48 @@
+import base64
+import io
+
+import torch
+import torch.nn.functional as F
+from PIL import Image
+
+from ts.torch_handler.image_classifier import ImageClassifier
+from ts.utils.util import map_class_to_label
+
+
+class ModelHandler(ImageClassifier):
+    def __init__(self):
+        super(ModelHandler, self).__init__()
+
+    def preprocess(self, data):
+
+        images = []
+        for row in data:
+            imgs = row.get("data") or row.get("body")
+            imgs = list(imgs.values())
+            for image in imgs:
+                if isinstance(image, str):
+                    # if the image is a string of bytesarray.
+                    image = base64.b64decode(image)
+
+                # If the image is sent as bytesarray
+                if isinstance(image, (bytearray, bytes)):
+                    image = Image.open(io.BytesIO(image))
+                    image = self.image_processing(image)
+                else:
+                    # if the image is a list
+                    image = torch.FloatTensor(image)
+
+                images.append(image)
```

Comment:
You can `torch.stack` images directly without an intermediate list

Reply:
+1

Reply:
How do we do this? torch.stack needs multiple elements of the same size? I copied this from the image_classifier handler